src/modules/navigation/hooks/helpers.ts

```diff
@@ -100,6 +100,11 @@ export const computePath = (
     case 'shortcut':
       return `/external/${file._id}`
     case 'directory':
+      // If we are in /favorites tab, we do not want it to appears in computed path
```

Comment:
`On mobile, if we are in /favorites tab...` I think will be better to understand next time what we are talking about here.

Reply:
Updated